src/css-skew-parsing.js

```diff
@@ -19,11 +19,16 @@
     if (angles.length != 1) {
       return null;
     }
+    var zeroAngle = new CSSAngleValue(0, 'deg');
     switch (type) {
       case 'x':
-        return [new CSSSkew(angles[0].degrees, 0), remaining];
+        var result = [new CSSSkew(angles[0], zeroAngle), remaining];
+        result[0].cssText = 'skewx(' + result[0]._ax.cssText + ')';
```

Comment:
We probably shouldn't be looking at private members or setting cssText (which will be a readonly member when implemented in Blink). It's setting a bad example for how you can use Typed OM (even though these are internal methods). Can we instead make a method on typedOM.internal that sets the cssText of a CSSStyleValue? (even if the method does the same thing I think it looks less weird).

Also in these cases, the cssText will be the consumed string right? I think it would be better to look at how many chars were consumed, and fetch the original parsed string, and use that (possibly lowercased), rather than constructing it again here.